cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,85 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import faker from 'faker';
 
-describe('Settings page', () => {
-  before(() => {
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
 
-  });
+describe('Settings page', () => {
+  let user;
 
   beforeEach(() => {
```

Comment:
You are logging in before each test. This is time-consuming and unnecessary for tests that don't require a logged-in user. Consider moving the login steps to only those tests that need it.

Reply:
But we need to have logged user for every test in 'Settings' testsuite. Or did you mean another testsuite?